src/node-lib/curriculum-api-2023/fixtures/curriculumUnits.fixture.ts

```diff
@@ -359,7 +375,7 @@ const curriculumUnitsTabFixture = (
         connection_future_unit_title: "Composition of numbers 6 to 10",
         connection_prior_unit_title: "Composition of numbers",
         domain: null,
-        domain_id: null,
+        domain_id: 17,
```

Comment:
This was intentional, as I had a bug that even though the feature was broken, all unit tests were still passing. I narrowed it down to needing at least one unit that had a domain_id. In summary - with this domain_id I believe it covers more edge cases.